docs/platform/release-notes/tizen-7-0-m1.md

```diff
@@ -0,0 +1,545 @@
+    - Version 23.3 has been upgraded to version 23.4.
+  - python3
+    - Version 3.9.1 has been upgraded to version 3.9.10.
+  - re2
+    - Version 20201101 has been upgraded to version 20220201.
+  - re2c
+    - Version 1.1.1 has been upgraded to version 3.0.
+  - rpmlint
+    - Version 1.4 has been upgraded to version 1.11.
+  - rsync
+    - Version 3.1.1 has been upgraded to version 3.2.3.
+  - sgml-skel
+    - Version 0.6 has been upgraded to version 0.7.1.
+  - sqlite
+    - Version 3.33.0 has been upgraded to version 3.36.0.
+  - sudo
+    - Version 1.9.4 has been upgraded to version 1_9_7p2.
+  - tzdata
+    - Version 2020a has been upgraded to version 2021e.
+  - util-linux
+    - Version 2.36 has been upgraded to version 2.37.
+
+
+### Application framework
+
+#### New and changed features
+
+- Tizen boot sequence
+  - A new feature has been ready to control app launch on system boot.
+  - A tool to check the status of the app after booting has been provided.
+- TIDL
+  - The TIDL compiler can generate new API set using the Cion-group API set.
+
+#### Fixes
+
+- App-core
+  - The implementation has been refactored into the C++ language.
+- Pkgmgr-info
+  - A cache for pkginfo-server has been applied to improve the online mode performance of getting package information.
+  - Offline mode performance of getting package information has been improved.
+- App-installer
+  - The preload package installation order in MIC builds has been changed for devices with low storage capacity.
+
+
+### Window and Interaction
+
+#### New and changed features
+
+- Open source
+  - wayland has been upgraded to version 1.20.0.
+  - The libxkb-common has been upgraded to version 1.3.1.
+  - The libinput has been upgraded to version 1.17.0.
+  - The libevdev has been upgraded to version 1.11.0.
+  - The mesa has been upgraded to version 21.2.3.
+- Enlightenment
+  - The Drag and Drop has been added. It provides wl_data_device_manager interface of wayland protocol to the wayland-clients.
+- Vulkan
+  - The vulkan-wsi-layer has been added. It implements Vulkan window system integration extensions.
+- OpenCL
+  - The Common OpenCL Packages have been added.
+    - The OpenCL-Headers has been added. It contains C language headers for the OpenCL API.
+    - The OpenCL-ICD-Loader has been added. It allows developers to build application against an Installable Client Driver loader.
+- TTS Framework
+  - In order to reduce memory usage, the TTS engine, which used to execute in multiple processes, has been improved to execute as one process.
+  - The client structure and TTS player to support thread safety has been improved.
+  - The TTS engine API to terminate the process has been added.
+  - The API to connect TTS engine synchronously has been added.
+  - The API to repeat TTS text has been added.
+- Sticker Framework
+  - C# sticker API has been added.
+- Text Input Framework
+  - Vietnamese and Thai language has been supported in the Tizen reference IME.

```

Comment:
```suggestion
  - Vietnamese and Thai language support have been added in the Tizen reference IME.

```

Reply:
will accept

Reply:
Made a slight change to the suggestion. Is this ok now?